mobile/src/src/components/Menu/MenuPage.jsx

```diff
@@ -1,41 +1,179 @@
-import React, { useContext } from "react";
-import { StyleSheet, Text, View, Button } from "react-native";
-import { AuthEndpoint } from "../../utils/endpoints";
-import AuthContext from "../../contexts/AuthContext";
+import React, { Component } from "react";
+import { StyleSheet, Text, View, TouchableOpacity, Image } from "react-native";
+import { Card, Paragraph, Avatar, IconButton } from "react-native-paper";
+import { ProfileEndpoint } from "../../utils/endpoints";
 import Header from "../common/Header";
+import MenuLogout from "./MenuLogout";
 
-const MenuPage = () => {
-  const auth = useContext(AuthContext);
+class MenuPage extends Component {
+  state = { user: [] };
```

Comment:
Dont think this needs to be an array, maybe make it an empty object? A little confusing to have it as an array
ie change to {}. Let me know if there is a specific reason

Reply:
okay sounds good, I changed it to {}